backends/dpdk/backend.cpp

```diff
@@ -71,6 +71,7 @@ void DpdkBackend::convert(const IR::ToplevelBlock *tlb) {
         new ConvertActionSelectorAndProfile(refMap, typeMap, &structure),
         new CollectTableInfo(&structure),
         new CollectAddOnMissTable(refMap, typeMap, &structure),
+        new CollectAddOnMissAdd(refMap, typeMap, &structure),
         new ValidateAddOnMissExterns(refMap, typeMap, &structure),
```

Comment:
This pass name looks weird. The next pass ValidateAddOnMissExterns also checks such constraints. Can we not  add these additional checks as well in that pass?

Reply:
Will check the ValidateAddOnMissExterns pass and get back.